docs/checklists/medtronic600.md

```diff
@@ -0,0 +1,236 @@
+# Pump Data direct from Medtronic 600-series pump
+
+## Checklist for Insulin Pump Implementation
+
+(Key:
+
+ - `[x]` available in data protocol/documented in spec and implemented
+ - `[-]` available in data protocol/documented in spec but *not* yet implemented
+ - `[?]` unknown whether available in data protocol/documented in spec; *not* yet implemented
+ - `*[ ]` TODO: needs implementation!
+ - `[ ]` unavailable in data protocol and/or not documented in spec and not yet implemented)
+
+### Required if Present
+
+#### Basals
+
+  - `[x]` scheduled basal
+    - `[x]` basal rate intervals with a start time, duration, and rate delivered
+    - `[x]` name of basal schedule on each scheduled basal rate interval
+    - `[?]` if basal schedule is a single (flat) rate all day, pump records a new basal rate interval every midnight
+  - `[x]` manual temp basal
+    - `[x]` basal rate intervals with a start time, duration, and rate delivered
+    - `[x]` object representing suppressed scheduled basal *for each segment of the basal schedule that the temp basal intersects*
+  - `[x]` percentage temp basal
+    - `[x]` basal rate intervals with a start time, duration, percent
+        - `[ ]` rate provided directly OR
+        - `[x]` rate computed from percent x suppressed.rate
+    - `[x]` object representing suppressed scheduled basal *for each segment of the basal schedule that the temp basal intersects*
+  - `*[ ]` "suspended" basals (see [status - suspends & resumes](#device-events) below)
```

Comment:
Do suspended and suppressed basals still need to be implemented?

Reply:
No, they're done. Will update document.